src/common/datomish/transact/bootstrap.cljc

```diff
@@ -38,6 +38,15 @@
                           :db/cardinality :db.cardinality/one}
    :db.alter/attribute   {:db/valueType   :db.type/ref
                           :db/cardinality :db.cardinality/many}
+
+   :db.schema/version    {:db/valueType   :db.type/long
+                          :db/cardinality :db.cardinality/one}
+
+   ;; unique-value because an attribute can only belong to a single
+   ;; schema fragment.
+   :db.schema/attribute  {:db/valueType   :db.type/ref
```

Comment:
This actually says that _schema fragments_ are related to attributes by the `:db.schema/attribute` relation.

Both schema fragments and attributes themselves are modeled as _entities_. An entity is a thing named by an `entid`. In this case, both area also identified by `ident`.

You can split up the world of Datomish into _values_ (5, "foo", 20161001T13:54:00Z), _entities_, and attributes that relate them. Attributes themselves are modeled as entities.